### DISCUSSION TITLE
2 side document

### DISCUSSION THREAD

**Discussion by Norbiros** [community/community#52233] [GitHub Education] [answered]
My document looks like this

_Moderators Edit: I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md)._

How I can submit it to github education? On one side I have my full name but on other duration information.

**Comment by samus-aran** [comment, marked as answer]
Hi @Norbiros 

We accept multiple pieces of evidence if you are able to take a photo of them along side each other.

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Let us know how you get on 👍

**Reply by Norbiros** [reply]
I already did it, but still thanks!

**Reply by samus-aran** [reply]
Awesome to hear! Welcome to GitHub Global Campus 🎉 

What offer or area of GitHub Global Campus are you most excited to try first? 🥇

**Reply by Norbiros** [reply]
IDK, can you recommend me something, please?

**Reply by samus-aran** [reply]
Sure!

- Connect with a local Campus Expert. For more information on campus experts, see "[About Campus Experts](https://education.github.com/students/experts)."
- Explore and claim offers for free industry tools from the [Student Developer Pack](https://education.github.com/pack).
- See upcoming in-person and virtual events for students, curated by GitHub Education and student leaders.
- View assignments from [GitHub Classroom](https://classroom.github.com/) with upcoming due dates.
- Stay in the know with [Campus TV](https://www.twitch.tv/githubeducation) episodes. Campus TV is created by GitHub and student community leaders and can be watched live or on demand.
- Discover and contribute to student-created repositories from GitHub Community Exchange. For more information, see "[About GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange)."

It's great to have you in the GitHub Education community and let us know what you enjoy 👋
